test/basics.js

```diff
@@ -13,77 +13,78 @@
  * limitations under the License.
  */
 
+'use strict'
 
-'use strict';
+const should = require('should')
+const describe = require('mocha').describe
+const it = require('mocha').it
+const nuid = require('../lib/nuid.js')
 
-const nuid = require('../lib/nuid.js');
-const should = require('should');
-
-function rangeEquals(ba, bb, start, end) {
-    var equal = true;
-    if (start === undefined) {
-        start = 0;
-    }
-    if (end === undefined) {
-        end = ba.length;
+function rangeEquals (ba, bb, start, end) {
+  var equal = true
+  if (start === undefined) {
+    start = 0
+  }
+  if (end === undefined) {
+    end = ba.length
```

Comment:
What happens if bb's length is smaller than ba's? The loop below will be bound to the length of ba (or whatever value is passed as `end`), but there will then be an attempt to get a value passed the length of the array, no?

Reply:
correct - this is just an assertion for the test